main.py

```diff
@@ -133,14 +133,11 @@ def run_scenario(current_scenario):
     current_scenario.preprocess_scenarios_data()
 
     # Train and eval on all partners according to scenario
-    start = timer()
     current_scenario.mpl = multi_partner_learning.init_multipartnerlearning_from_scenario(
         current_scenario,
         is_save_fig=True,
     )
-    current_scenario.compute_mpl()
-    end = timer()
-    current_scenario.federated_computation_time_sec = end - start
```

Comment:
I think we should revert this change. The timer created inside the `compute_mpl()` method will conflict with the timers created when doing contributivity estimation, for instance here --> https://github.com/SubstraFoundation/distributed-learning-contributivity/blob/8e5a242d150bf7e73853d9377984295047d59474/contributivity.py#L149

Reply:
Hmm, interesting. Don't you think the respective variable scopes isolate `start` and `end`?

Reply:
Ah no, I get what you mean. The result stored as a `Scenario` attribute will be overwritten each time `compute_mpl()` is called within a contributivity approach.

Reply:
Actually I think it won't: the result stored as a `Scenario` attribute is from the `MultiPartnerLearning` object that is itself a `Scenario` attribute. When a contributivity approach is called, and method `not_twice_characteristic()`, a dedicated `MultiPartnerLearning` object is instantiated. I don't see where there would be a conflict. Do you still do @RomainGoussault ?

Reply:
Actually I think you are right. I thought timer() was doing some complex stuff, but it's not the case.